wdl2cwl/main.py

```diff
@@ -139,25 +132,6 @@ def get_cwl_docker_requirements(
     return cwl.DockerRequirement(dockerPull=dockerpull)
 
 
-def get_literal_name(
```

Comment:
This function no longer serves it's main purpose since we started tracking non_static inputs.